### ISSUE TITLE
Exception thrown when run dotnet-projectmodel-server

### ISSUE THREAD

**Issue by troydai** [dotnet/cli#1049]
```
Unhandled Exception: System.IO.FileLoadException: Could not load file or assembly 'System.Runtime.Extensions, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)
   at Microsoft.Extensions.Logging.LoggerFactory..ctor()
   at Microsoft.DotNet.ProjectModel.Server.Program.<>c__DisplayClass8_0.<Main>b__0()
   at Microsoft.Dnx.Runtime.Common.CommandLine.CommandLineApplication.Execute(String[] args)
   at Microsoft.DotNet.ProjectModel.Server.Program.Main(String[] args)
```

**Comment by piotrpMSFT** [comment]
#1004 should resolve this when fixed. We've normalized Standard Library references.

**Comment by danroth27** [comment]
@troydai @piotrpMSFT #1004 is now merged. Should this be closed?

**Comment by troydai** [comment]
This one is fixed along with #1004.
